src/lib/app-provider/hooks/useBaseApiRoute.ts

```diff
@@ -10,6 +10,7 @@ export const useBaseApiRoute = (
     | "codes"
     | "accounts"
     | "rest"
+    | "feeTokens"
```

Comment:
fee_tokens

Reply:
Perhaps we could consider using `native_tokens`.